### ISSUE TITLE
Facing some issues.

### ISSUE THREAD

**Issue by WantToMakeNps** [Unovamata/Neopets-Shop-And-Attic-Autobuyer-Cracked#48]
Hi,
Thanks for making this project.
I tried installing and using this because trying to rs anything is basically impossible these days :( .

I just encountered one issue:
I have put default profit at 10k
and other at 500%
I assume this means if there's a profit of 10k or 5x the amt of what I've bought at it would purchase the item.

However the aber has been only buying mac and cheeses.

Can you help me with what I'm doing wrong.

Also it mentions I should open the item database of aber.
So just keeping it open is ok or should you have it open to a specific page where what you want to buy is present?

I have set the default profit to 
![image](https://github.com/Unovamata/Neopets-Shop-And-Attic-Autobuyer-Cracked/assets/91705840/06bbf76a-5bc9-4c67-8e4e-7aa03ed0ec62)
![image](https://github.com/Unovamata/Neopets-Shop-And-Attic-Autobuyer-Cracked/assets/91705840/13020233-79cb-4db7-ac0d-1c738d037263)
![image](https://github.com/Unovamata/Neopets-Shop-And-Attic-Autobuyer-Cracked/assets/91705840/d4af16e5-549b-4dc7-8917-50d6f6627955)

Thank you for your help.

**Screenshots edited to protect the integrity and anonymity of the user's account.**

**Comment by Unovamata** [comment]
Hi WantToMakeNPs I hope you have been doing well. Welcome to NeoBuyer+ it means a lot to me for users to find value in what am I developing.

As for your issue. I noticed you have been using the database for purchases. Currently the database has not been updated ever since August 2023. Since I had to update this system manually every week it became unsustainable very fast. 

I am currently working on a way to solvent this issue but at the moment, navigate to the [Restock List Generator](https://github.com/Unovamata/Neopets-Shop-And-Attic-Autobuyer-Cracked/wiki/Restock-List-Generator) page to use and understand how to create your custom restock lists.

Thank you for your interest in NeoBuyer+. I hope you continue to use NeoBuyer+ for the foreseeable future! 😄
